docs/manage/data/label.md

```diff
@@ -38,6 +38,10 @@ Click on the image where you would like to add the bounding box and drag to wher
 
 {{<gif webm_src="/manage/data/label-dog.webm" mp4_src="/manage/data/label-dog.mp4" alt="Add a bounding box around the dog in an image">}}
 
+If you want to see the image in bigger, click on the expand side menu arrow:
```

Comment:
I think "in bigger" sounds wrong
```suggestion
If you want to see the image bigger, click on the expand side menu arrow.
```
or
```suggestion
If you want to expand the image, click on the expand side menu arrow.
```
maybe both this and the other one should either have a colon or a period?

Reply:
```suggestion
If you want to expand the image, click on the expand side menu arrow:
```